src/screens/ParadeMapScreen/Map.js

```diff
@@ -110,6 +123,14 @@ class Map extends Component<Props, State> {
     }
   };
 
+  handleMarkerPress(stage: Event) {
+    this.setState({ tileDetails: stage, activeMarker: stage.id });
+  }
+
+  handleMapPress = () => {
+    this.setState({ tileDetails: null, activeMarker: null });
```

Comment:
I think we should check here if `tileDetails` and `activeMarker` are already null and don't call `setState` in this case. [Reason](https://reactjs.org/docs/react-component.html#setstate):

> `setState()` will always lead to a re-render unless `shouldComponentUpdate()` returns `false`.

Reply:
Hm, or maybe it's fine. That's really hard to understand from the documentation. I think we would at least have to make `Map` a `PureComponent`.

Reply:
I'm confused, you would like me to make Map a PureComponent?